README.md

```diff
@@ -264,6 +264,122 @@ Alby enforces [Conventional Commits Specification](https://www.conventionalcommi
 
 Alby uses [Weblate](https://weblate.org/en/) to manage translations for different locales. If you'd like to contribute, you can [add translations here](https://hosted.weblate.org/projects/getalby-lightning-browser-extension/getalby-lightning-browser-extension/).
 
+#### Rules for developers adding new i18n translation strings:
+
+[Not to be confused with language translations]
+
+**Use underscores instead of camelCase.**
```

Comment:
You can use with the headings here as well. i.e.
`##### Heading`

No need to end headlines with a `.`